src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -261,19 +245,17 @@ class OrganizationIntegrations extends AsyncComponent<
         />
       );
     }
-
-    return (
-      <SentryAppInstallationDetail
-        key={`sentry-app-row-${app.slug}`}
-        data-test-id="integration-row"
```

Comment:
@NisanthanNanthakumar we should keep this`data-test-id` since I think it's used in tests